beezee/assetlist.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "beezee",
+  "assets": [
+    {
+      "description": "BeeZee native blockchain",
+      "denom_units": [
+        {
+          "denom": "ubze",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "bze",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "ubze",
+      "name": "BeeZee",
+      "display": "bze",
+      "symbol": "BZE",
+      "logo_URIs": {
+	      "png": "https://raw.githubusercontent.com/bze-alphateam/Official-BZEdge-Graphics/master/Digital-Web/icon/Color/BZEdge_Color_icon.png"
```

Comment:
the issue is that the image URI is still pointing outside of this repo. It's should look more like: "https://raw.githubusercontent.com/cosmos/chain-registry/master/beezee/images/bze.png"

Reply:
Gotcha! Thanks! changed it.